lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2305,6 +2324,7 @@ def main():
         customization_spec=dict(type='str', default=None),
         vapp_properties=dict(type='list', default=[]),
         datastore=dict(type='str'),
+        convert=dict(type='str'),
```

Comment:
Add documentation for this parameter with `version_added`

Reply:
ok is commit